package.json

```diff
@@ -98,9 +98,9 @@
     ]
   },
   "dependencies": {
-    "@azure/msal-common": "^14.13.0",
-    "@azure/msal-node": "^2.10.0",
-    "@ministryofjustice/frontend": "^2.1.1",
+    "@azure/msal-common": "^14.14.2",
+    "@azure/msal-node": "^2.13.0",
```

Comment:
Upgrade to msal-node v2.13.0 breaks the application with the following error:- 
```
[Node] TypeError: Cannot read properties of undefined (reading 'LogLevel')
[Node]     at Object.<anonymous> (/Users/.../Projects/laa-crime-equinity-historical-data-frontend/dist/server/auth/authConfig.js:27:43)
[Node]     at Module._compile (node:internal/modules/cjs/loader:1376:14)
[Node]     at Module._extensions..js (node:internal/modules/cjs/loader:1435:10)
[Node]     at Module.load (node:internal/modules/cjs/loader:1207:32)
[Node]     at Module._load (node:internal/modules/cjs/loader:1023:12)
[Node]     at Module.require (node:internal/modules/cjs/loader:1235:19)
[Node]     at Module.patchedRequire (/Users/.../Projects/laa-crime-equinity-historical-data-frontend/node_modules/diagnostic-channel/dist/src/patchRequire.js:16:46)
[Node]     at Hook._require.Module.require (/Users/.../Projects/laa-crime-equinity-historical-data-frontend/node_modules/require-in-the-middle/index.js:188:39)
[Node]     at require (node:internal/modules/helpers:176:18)
[Node]     at Object.<anonymous> (/Users/.../Projects/laa-crime-equinity-historical-data-frontend/dist/server/auth/authProvider.js:31:22)
```

Need to investigate this before upgrading library.